app/src/main/cpp/WidgetPlacement.h

```diff
@@ -37,10 +37,12 @@ struct WidgetPlacement {
   int32_t GetTextureWidth() const;
   int32_t GetTextureHeight() const;
 
+  static const float WORLD_DPI_RATIO;
```

Comment:
nit: I would prefer we reserve all caps for `#define` macro and use camel case for variables even `const` ones.